.env.defaults

```diff
@@ -19,8 +19,6 @@ GW_HOST=''
 GW_USERNAME=default
 GW_PASSWORD=''
 GW_TLS=true
-GW_ENABLE_OFFLINE_QUEUE=true
-GW_LAZY_CONNECT=true
```

Comment:
This envs are not used anywhere, so I removed them

Reply:
are you sure? https://github.com/warp-contracts/warp-dre-node/blob/main/src/config.js#L36-L37

Reply:
Yep cause, `config.enableOfflineQueue` is not used later and same parameter exists for `bullmqConfig` so i think it is some kind of "chochlik" here :)

Reply:
Maybe I'm missing sth, but what about https://github.com/warp-contracts/warp-dre-node/blob/main/src/listener.js#L281 ? The configuration object is passed into the Redis instance.

This configuration is used when creating the Redis client for subscribing for notifications from the Warp Gateway (about new interactions and new contracts).

The `bullmqConfig` in related to the internal Redis configuration for the bullmq.

Reply:
Ohh totally true, my bad, thanks for catching this! 
Fixed